java/opendcs/src/main/java/decodes/cwms/resevapcalc/ResEvap.java

```diff
@@ -241,23 +241,23 @@ public boolean compute(Date currentTime,
                             "                       m/s    deg C     %      mb      deg C           m/s                m      ********** W/m**2 ***********************     mm/d";
                     metout.write(heading);
                     metout.newLine();
-                    }
-                } catch (IOException ex)
-                {
-                LOGGER.info("Unable to read {}", metoutfil.getAbsolutePath(), ex);
                 }
+            } catch (IOException ex)
+            {
+                LOGGER.info("Unable to read {}", metoutfil.getAbsolutePath(), ex);
```

Comment:
Use `LOGGER.atInfo().setCause().log("...", parameter)`

The used version of .info may not use ex as an actual exception.